website/templates/index.html

```diff
@@ -26,7 +26,7 @@ <h1 class="page-header">{% trans "Find Bugs, Win Points and Prizes" %}</h1>
     <div class="container">
         <center>
             <div class="row">
-                <div class="col-sm-4">    
+                 <div class="col-sm-3 cborder">    
```

Comment:
Just in case:
instead of making a new commit, use: `git commit --amend --no-edit`
And then push using `git push upstream branch_name -f`